unionml/fastapi.py

```diff
@@ -49,16 +43,15 @@ def root():
     @app.post("/predict")
     async def predict(
         inputs: Optional[Union[Dict[str, Any], BaseModel]] = Body(None),
-        features: Optional[List[Dict[str, Any]]] = Body(None),
+        features: Optional[Union[List[Dict[str, Any]], str]] = Body(None),
```

Comment:
how hard would it be to add a test for this change?

Reply:
ah, I didn't mean to check this change in, going to revert